lib/galaxy/model/__init__.py

```diff
@@ -6346,3 +6358,24 @@ def _prepare_metadata_for_serialization(id_encoder, serialization_options, metad
         processed_metadata[name] = value
 
     return processed_metadata
+
+
+def objects_requiring_history_updates(iter):
```

Comment:
Can you use a different variable name? `iter` is a standard library function: https://docs.python.org/3/library/functions.html#iter

Reply:
Done in the rebase, thanks @nsoranzo!